cal-itp/calitp.org
# Add new Press, Resources and Data Plans tag

closes #203 
closes #225 

## What this PR does
- Adds new resources and press for GTFS, Contactless Payments, Benefits and Data Plans
- Corrects formatting of `Mobility Data` press outlet
- Corrects existing bug of the Resources page showing an extra `|` before the tags
- Corrects the formatting for a lot of the dates, so that the articles are actually correctly ordered by date

## How to test/review
Use the filters on this Google Spreadsheet (https://docs.google.com/spreadsheets/d/1xJpgr8JBG9TgZF2mHB8Vjnhu453No4dHoNjFIhb0FQw/edit#gid=0) - Filter by Press or Resource page, and then by Tag, to ensure the results and content are the same.
